docs/antrea-ipam.md

```diff
@@ -32,7 +32,7 @@ controller. Default is false.
 
 - `clusterCIDRs`: CIDR Ranges for Pods in cluster. String array containing single
 CIDR range, or multiple ranges. The CIDRs could be either IPv4 or IPv6. Example
```

Comment:
We should mention only one CIDR of an IP family can be used, and we should validate it in controller Options.Validate().

Reply:
And in this case, do you think we should consider two parameters for v4 and v6 respectively? Or we are thinking about future extension?

Reply:
Done.
I think the parameters are following kube-controller-manager style where a single parameter is used to take CIDRs for two IP families. And yes, it's possible to extend it to support multiple CIDRs per IP family.